### ISSUE TITLE
Why we are using fit_transform when doing the MinMax Scaler and StandardScaler on training set and then while performing the same thing on test set we tend to use the .transform method..

### ISSUE THREAD

**Issue by AMEERKOTTA** [AMEERKOTTA/Complete-ML-Course-with-Python---Udemy#4]
Why we are using fit_transform when doing the MinMax Scaler and StandardScaler on training set and then while performing the same thing on test set we tend to use the .transform method..

Explain with Examples
